### PR TITLE
openeo GRASS driver: continue linting

### PR DESCRIPTION
**Pull request by metzm**
* fix F821

### CODE REVIEW

**Comment by metzm** [comment]
In ` tests/test_process_graph_validation.py` are some deactivated tests, but the corresponding python code has not been commented out or removed, causing F821 errors. Any suggestion what to do with these tests?

**Comment by mmacata** [comment]
> In ` tests/test_process_graph_validation.py` are some deactivated tests, but the corresponding python code has not been commented out or removed, causing F821 errors. Any suggestion what to do with these tests?

Is it deprecated code for an older API version? They seem to succeed if some deprecated code is not available. So for me it seems to be a solution while upgrading to the new api version and can be removed now.

File: src/openeo_grass_gis_driver/actinia_processing/aggregate_spatial_process.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from random import randint
```
[the remark this answers is not in the document]
Comment (metzm) [reply src/openeo_grass_gis_driver/actinia_processing/aggregate_spatial_process.py:2]: `randint` is used in aggregate_spatial_process.py:L120

**Comment by metzm** [comment]
> > In ` tests/test_process_graph_validation.py` are some deactivated tests, but the corresponding python code has not been commented out or removed, causing F821 errors. Any suggestion what to do with these tests?
> 
> [...] for me it seems to be a solution while upgrading to the new api version and can be removed now.

OK, removed.
The only errors left are now `E501 line too long`

File: src/openeo_grass_gis_driver/actinia_processing/aggregate_spatial_process.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mmacata) [reply src/openeo_grass_gis_driver/actinia_processing/aggregate_spatial_process.py:2]: Now i see it. Strange that it was overlooked by the linter (or me).

**Pull request merged**